package.json

```diff
@@ -94,6 +94,7 @@
 		"@babel/cli": "7.18.9",
 		"@babel/core": "7.18.9",
 		"@babel/plugin-proposal-class-properties": "7.18.6",
+		"@babel/plugin-proposal-optional-chaining": "^7.20.7",
```

Comment:
I noticed all the packages we added are using _compatible with version_ prefix (`^`), while all the WooCommerce packages use fixed versions.

We should respect that, although it would be nice to know what were the reasons that led them to fix all versions 🙂

Reply:
My mistake, most of the packages use fixed versions, but not all. So forget what I said.